assets/schema_input.json

```diff
@@ -31,30 +35,19 @@
                 ]
             },
             "status": {
-                "errorMessage": "Status can only be 0 or 1",
-                "anyOf": [
-                    {
-                        "type": "string",
-                        "pattern": "^(0|1)*$"
-                    },
-                    {
-                        "type": "string",
-                        "maxLength": 0
-                    }
-                ]
+                "type": "integer",
+                "errorMessage": "Status can only be 0 or 1. Defaults to 0, if none is supplied.",
+                "meta": ["status"],
+                "default": 0,
+                "minimum": 0,
+                "maximum": 1
             },
             "lane": {
-                "errorMessage": "Lane cannot contain spaces",
-                "anyOf": [
-                    {
-                        "type": "string",
-                        "pattern": "^\\S+$"
-                    },
-                    {
-                        "type": "string",
-                        "maxLength": 0
-                    }
-                ]
+                "type": "string",
+                "pattern": "^\\S+$",
+                "unique": ["patient", "sample"],
+                "dependentRequired": ["fastq_1"],
+                "meta": ["lane"]
             },
             "fastq_1": {
```

Comment:
can we add patterns to ensure it finishes with `{fastq,fq}.gz`?

Reply:
forget I said anything

Reply:
we could add it if you need it